docs/getting_started.rst

```diff
@@ -26,13 +43,17 @@ In ``fast_pauli``, we represent pauli matrices using the ``Pauli`` class. For ex
     pauli_y = fp.Pauli('Y')
     pauli_z = fp.Pauli('Z')
 
+    str(pauli_0)  # returns "I"
+
 We can also multiply two ``Pauli`` objects together to get a ``Pauli`` object representing the tensor product of the two pauli matrices.
```

Comment:
I think you meant matrix product here. you get a pair of phase and pauli object after @ing two Pauli objects